### ISSUE TITLE
Error: Compute CTC targets failed!

### ISSUE THREAD

**Issue by meetyogi98** [tesseract-ocr/tesseract#3339]
------------------------

### Environment

* **Tesseract Version**:tesseract 4.1.1
* **Commit Number**: 
* **Platform**: Ubuntu 16.04 64 bit

### Current Behavior:
Loaded file ./100_verygood_preprocessed/training/eng.lstm, unpacking...
Warning: LSTMTrainer deserialized an LSTMRecognizer!
Continuing from /home/petpooja/Downloads/100_verygood_preprocessed/training/eng.lstm
Appending a new network to an old one!!Warning: given outputs 111 not equal to unicharset of 105.
Num outputs,weights in Series:
  Lfx256:256, 394240
  Fc105:105, 26985
Total weights = 421225
Built network:[1,0,0,1[C5,5Ft16]Mp3,3Lfys64Lfx128Lrx128Lfx256Fc105] from request [Lfx256 O1c111]
Training parameters:
  Debug interval = 0, weights = 0.1, learning rate = 0.001, momentum=0.5
null char=104
Loaded 1751/1751 lines (1-1751) of document /home/petpooja/Downloads/100_verygood_preprocessed/training/rough_1.lstmf
Loaded 1751/1751 lines (1-1751) of document /home/petpooja/Downloads/100_verygood_preprocessed/training/rough_1.lstmf
Compute CTC targets failed!
Compute CTC targets failed!
Compute CTC targets failed!
Blank transcription:  
At iteration 100/100/104, Mean rms=5.606%, delta=39.451%, char train=118.761%, word train=100%, skip ratio=4%,  New worst char error = 118.761 wrote checkpoint.

Blank transcription:  
Compute CTC targets failed!
Compute CTC targets failed!
Compute CTC targets failed!
Blank transcription:  
At iteration 200/200/209, Mean rms=5.412%, delta=36.192%, char train=109.469%, word train=100%, skip ratio=4.5%,  New worst char error = 109.469 wrote checkpoint.

Blank transcription:  
Blank transcription:  
Compute CTC targets failed!
Compute CTC targets failed!
Compute CTC targets failed!
Compute CTC targets failed!
Compute CTC targets failed!
Blank transcription:  
Blank transcription:  
Compute CTC targets failed!
Blank transcription:  
At iteration 300/300/320, Mean rms=5.305%, delta=33.814%, char train=106.337%, word train=100%, skip ratio=6.667%,  New worst char error = 106.337 wrote checkpoint.

Compute CTC targets failed!
Compute CTC targets failed!
Compute CTC targets failed!
Compute CTC targets failed!
Blank transcription:  
Compute CTC targets failed!
Blank transcription:  
At iteration 400/400/427, Mean rms=5.286%, delta=33.704%, char train=104.733%, word train=100%, skip ratio=6.75%,  New worst char error = 104.733 wrote checkpoint.

Compute CTC targets failed!
Blank transcription:  
At iteration 499/500/529, Mean rms=5.287%, delta=33.733%, char train=103.889%, word train=100%, skip ratio=5.8%,  New worst char error = 103.889 wrote checkpoint.

Finished! Error rate = 100

### Expected Behavior:
How can I remove such error?

**Comment by stweil** [comment]
There exist already several identical issue reports which explain this. Typically those messages indicate bad training data, for example multi-line images instead of single-line images or vertical instead of horizontal text. The latest Tesseract reports the files which cause the problem. The training process will simply ignore that data, in your case 5.8 % of your data are skipped.

**Comment by meetyogi98** [comment]
Thank you for your reply.

I also want to know, how did you get to know that I have 5.8% of such data?

**Comment by stweil** [comment]
That's only an estimate, see `skip ratio` in the log messages:

     At iteration 499/500/529, Mean rms=5.287%, delta=33.733%, char train=103.889%, word train=100%, skip ratio=5.8%, New worst char error = 103.889 wrote checkpoint.

**Comment by meetyogi98** [comment]
Got it
Thanks a lot.
